cfpb/cfgov-refresh
# Update CCDB landing page

Update the content on the CCDB landing page

## Additions

- Added a feature flag to keep the current CCDB landing page content visible until it's time to make the updates live

## Changes

- Updated content on the CCDB landing page

## Testing

1. Visit http://localhost:8000/data-research/consumer-complaints/ in your browser of choice
2. Confirm that the page looks exactly like [what's in production](https://www.consumerfinance.gov/data-research/consumer-complaints/) with the exception of the minor fix of "like" to "such as" in the second paragraph of the "Download the data" section (which can go live at any time)
3. Enable the `CCDB_SEPT_2019_UPDATES` feature flag in the Wagtail admin
4. Confirm that the CCDB landing page now looks like the screenshot below

## Screenshots

![ccdb-updates](https://user-images.githubusercontent.com/1862695/64811133-af8cca00-d56a-11e9-8a17-4f0d93a6ce26.png)

## Notes

- This page is set to go live at a specific time, so we'll enable the feature flag to show the updates when that time comes.
- To stay consistent with how other feature flags are used on this page, I'm using the new `CCDB_SEPT_2019_UPDATES` to set a context variable instead of using the flag directly in the Django template.

## Todos

- After the updates have launched, we should remove the feature flag and clean up the template so that it has only the new content.

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
